test/services/smart-fields-values-injector.test.js

```diff
@@ -1,53 +1,100 @@
+// eslint-disable-next-line max-classes-per-file
```

Comment:
[OPINION] We can remove this line if we something else than classes. I think that using simple objects could be enough

Reply:
I have to use class because I need to update its prototype.
Modifying the prototype of `Object` could lead to prototype pollution, that's why I'd like to keep it away :)

Reply:
Okay, nice and clear